src/components/button/button.tsx

```diff
@@ -57,18 +57,24 @@ export class Button {
     @Prop({ reflect: true })
     public loading = false;
 
-    @Element()
-    private limelButton: HTMLLimelButtonElement;
+    /**
+     * Set to `true` to indicate failure instead of success when the button is
+     * no longer in the `loading` state.
+     */
+    @Prop()
+    public loadingFailed = false;
+
+    private justLoadedTimeout?: number;
 
-    private justLoadedTimeout: NodeJS.Timeout;
+    private justLoaded = false;
```

Comment:
I forgot to add the `@State()` decorator here. When testing manually I saw the `just-loaded` class wan't removed, so it means I made some mistake when refactoring the tests

Reply:
fixed now! (and updated the tests to fail when it should)